Exception creating metadata for service with console

An exception occurs if non existing metadata are created with the deegree console. 

Exception in tomcat log:
```
SCHWERWIEGEND: Error Rendering View[/console/generic/xmleditor.xhtml]
javax.el.ELException: /console/generic/xmleditor.xhtml @45,102 value="${xmlEditorBean.content}": Error reading 'content' on type org.deegree.console.generic.XmlEditorBean
	at com.sun.faces.facelets.el.TagValueExpression.getValue(TagValueExpression.java:114)
	at javax.faces.component.ComponentStateHelper.eval(ComponentStateHelper.java:194)
	at javax.faces.component.ComponentStateHelper.eval(ComponentStateHelper.java:182)
	at javax.faces.component.UIOutput.getValue(UIOutput.java:170)
...
Caused by: java.net.MalformedURLException: no protocol: null
	at java.net.URL.<init>(URL.java:593)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at org.deegree.console.generic.XmlEditorBean.getContent(XmlEditorBean.java:185)
```
Steps to reproduce the exception:

1. Create a new workspace
2. Activate the new workspace
3. Create a new service (WFS, minimal)
4. Click on **Edit Metadata** 

Instead of the editor an exception page occurs. A file is not created.

--- a/deegree-services/deegree-webservices/src/main/java/org/deegree/console/webservices/ServiceConfig.java
+++ b/deegree-services/deegree-webservices/src/main/java/org/deegree/console/webservices/ServiceConfig.java
@@ -45,7 +45,7 @@ import org.deegree.workspace.ResourceMetadata;
 
 public class ServiceConfig extends Config {
 
-    private static final URL METADATA_EXAMPLE_URL = ServicesBean.class.getResource( "/META-INF/schemas/services/metadata/3.2.0/example.xml" );
+    private static final URL METADATA_EXAMPLE_URL = ServicesBean.class.getResource( "/META-INF/schemas/services/metadata/3.4.0/example.xml" );
 
     public ServiceConfig( ResourceMetadata<?> metadata, ResourceManager<?> resourceManager ) {
         super( metadata, resourceManager, "/console/webservices/index", true );

